sktime/forecasting/tests/test_all_forecasters.py

```diff
@@ -931,7 +931,16 @@ def test_pridect_signature(self, estimator_class):
         assert "X" in sig.parameters.keys()
         assert "y" in sig.parameters.keys()
 
+    def _check_global_tag(self, estimator_instance):
```

Comment:
where is this coming from, why are we changing the tests?

Best practice is to not modify the interface tests while adding an object of an interface.

Reply:
Source: https://github.com/sktime/sktime/pull/6686
These editions are made in all global forecasting PRs, as it is important and avoids certain test cases that should not be performed on global estimators.

Reply:
It's becasue the `broadcasting` parameter. If `broadcasting==True`, the global forecasting ability will be turned off in `__init__`. So some of the `estimator_instance`s do not have the global forecasting ability.